eslzArm/eslz-portal.json

```
@@ -2749,6 +2749,67 @@
                             "visible": true
                         }
                     ]
+                },
+                {
+                    "name": "decommissionedSandboxZones",
+                    "label": "Decommissioned/Sandbox",
+                    "subLabel": {
+                        "preValidation": "",
+                        "postValidation": ""
+                    },
+                    "bladeTitle": "ALZ - Decommissioned and Sandbox Landing Zones Settings",
+                    "elements": [
+                        {
+                            "name": "infoBox2",
+                            "type": "Microsoft.Common.InfoBox",
+                            "visible": true,
+                            "options": {
+                                "text": "You can optionally choose to enable the assignment of recommended policies for the Decommissioned and Sandbox landing zones that will enable policies according to best practices.",
+                                "uri": "https://docs.microsoft.com/azure/cloud-adoption-framework/ready/enterprise-scale/design-principles#policy-driven-governance",
```

```suggestion
                                "uri": "https://aka.ms/alz/policies",
```

Should we link to this instead and expand whats included on that page with more info about the specific policies in the initiative?

---

@jtracey93 I do agree, but the whole policies document needs to be reworked. Currently it contains really unhelpful content. Agree to remove the version column, but we also need to better document what the initiative/policy does, perhaps why it's included, etc.